### PR TITLE
Makes meteor shields cheaper, and lower on the tech tree

### PR DESCRIPTION
**Pull request by chefferz**
Want to keep the meteor shields at 1 minute? Then this is the balance to keeping them at just that.

Current rate of tech points usage makes everything but fringe things like this actually purchased unless it's needed. This PR sets the meteor shields to base tier, and allows them to be purchased off the bat so engineering can preemptively set them up.

Cost is lowered significantly (about 66%)
Tech point is only 2k and at basic engineering tier rather than behind a much larger amount of tiers before you can unlock it.

I encourage you to read the discussions from the actual engineering mains on the last PR pertaining this,

### CODE REVIEW

**Comment by PersianXerxes** [comment]
Good PR, if we won't revert the overly nerfed timers.

Meteors are **not** a well designed event and repairing meteor damage is not fun or interesting as an engineer, nor can it be made so.
This allows meteor shields to be set up preemptively, which is what keeps being suggested when people point out the fact that the one minute timer is not enough.
It's beyond unreasonable to keep a one minute meteor timer and not make meteor satellites easier to preemptively build.

File: code/modules/research/designs/misc_designs.dm [hunk 1]
```diff
@@ -532,7 +532,7 @@
 	desc = "A blue print of a early model of the Meteor defence turret."
 	id = "meteor_defence"
 	build_type = PROTOLATHE
-	materials = list(MAT_METAL = 50000, MAT_GLASS = 50000, MAT_SILVER = 8500, MAT_GOLD = 8500, MAT_TITANIUM = 7500, MAT_URANIUM = 7500)
+	materials = list(MAT_METAL = 20000, MAT_GLASS = 20000, MAT_SILVER = 2500, MAT_GOLD = 2500, MAT_TITANIUM = 2500, MAT_URANIUM = 2500)
```
Comment (Trilbyspaceclone) [inline code/modules/research/designs/misc_designs.dm:535]: Dear fucken god why are these so cheap now?
5 metal 5glass 1.4 silver 1.4 gold 1.4 titanium 1.4 uranium at t1
Thats so fucken cheap. 
At max parts this is so fucken cheap you can mass make these without issue.
Thats not what these are for. Buy them from cargo shift start if you want to protect the station.
When these were made it was for the sole reason of `OH shot meteors, quick protect the SM/Armor/Grav gen`
Not the fucken full station
Sci meteor shields also half less range then cargo's

**Comment by Trilbyspaceclone** [comment]
Dear fucken god the meteor shield power creep in this is unbearably cheap, 2k points to save over 12k in credits well now it being /even easyer to get almost round start/ is flat out unreasonable in a balance scene and dosnt fix the issue of no one building them and when they are flat out made they make meteors LESS likely to happen making this will just make meteors less likely to happen in the first place.
So you are both making the meme of printing/making sci meteors for no event EVEN better then what they were, a costly way to protect some area's of the station into a new "Just build them every shift to lower meteor odds before they can even happen"

**Comment by silicons** [comment]
Too strong of powercreep. Meteor shields being added to R&D for a high cost was a way to make it balanced as they used to be orderable only. This is just terrible, the only thing unreasonable is expecting meteors to be made even more trivial.

I'm fully aware engineering mains supported the longer timers. I'm fully aware that the people who commented do not like fixing meteor damage. But to me, this is an expectation you are signing up for when you sign up to do engineering work. If it's so unreasonable to set up meteor shields beforehand and you hate fixing damage so much (when most waves aren't catastrophic) don't play engineering.

Also here time for me to be rude: Don't be that guy who implies you have hard power over what happens regarding meteors, or really, anything. This is a tough decision for everyone involved because yes I am aware of how interesting a bunch of rocks blowing up the station because the computer rolled a bad roll is, but you continuously adopting a tone that implies you have any authority to be forcing changes here is not winning you anything.

Regardless, I'm not closing this anytime soon as I want to see the effects from the two meteor PRs I have opened to lessen the amount of misdirection regarding timer and the effects of directional meteors.

**Comment by GrayRachnid** [comment]
😂

**Pull request closed without merging**